lib/opencensus/stats/measure.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+
+require "opencensus/stats/measurement"
+
+module OpenCensus
+  module Stats
+    # Measure
+    #
+    # The definition of the Measurement.
+    class Measure
+      # Describes the unit used for the Measure.
+      # Should follows the format described by
+      # http://unitsofmeasure.org/ucum.html
+      # Unit name for general counts
+      # @return [String]
+      UNIT_NONE = "1".freeze
+
+      # Unit name for bytes
+      # @return [String]
+      BYTE = "By".freeze
+
+      # Unit name for Kilobytes
+      # @return [String]
+      KBYTE = "kb".freeze
+
+      # Unit name for Seconds
+      # @return [String]
+      SEC = "s".freeze
+
+      # Unit name for Micro seconds
+      # @return [String]
+      MS = "ms".freeze
+
+      # Unit name for Neno seconds
+      # @return [String]
+      NS = "ns".freeze
+
+      # @return [String]
+      attr_reader :name
+
+      # @return [String]
+      attr_reader :description
+
+      # Unit type of the measurement. i.e "kb", "ms" etc
+      # @return [String]
+      attr_reader :unit
+
+      # Data type of the measure.
+      # @return [String]
+      attr_reader :type
```

Comment:
Document the list of possible values. Also, I suspect the type is supposed to be `Symbol` rather than `String`.

Reply:
Fixed.